config/prod/dgutman-dsa-server-prod-ec2.yaml

```diff
@@ -0,0 +1,46 @@
+# Provision an EC2 instance
+template_path: "remote/managed-ec2-linux-v2.j2"
+stack_name: "dgutman-dsa-server-prod-ec2"
+sceptre_user_data:
+  Distro: "ubuntu"     # (valid values: aws, ubuntu) AMIId must match distro
+  # (Optional) Expose ports to incoming traffic (default is no open ports) (valid range: 1-65535)
+  OpenPorts: ["80","443", "22"]
```

Comment:
Does this web server need 80, or does this just for a redirect?

Reply:
Here's the response I got from our collaborator:

> For Redirect and for initial certificate grabbing with certbot

Reply:
@ahayden regarding data governance - this instance will only contain imaging data
- these data are not access-controlled (i.e. they are to be publicly released)
- all HTAN data that are not access-controlled have, by grant contract, been de-identified by data contributors (i.e. no PHI)
- PHI data can leak only accidentally (e.g. image header metadata left over by a data contributor); we are not legally liable if that happens. Even though at the HTAN DCC we are not required, we do at present review images (and accompanying metadata) for PHI prior to public data release.

In all these respects, this new instance will host the same data as the instance hosting the existing DSA deployment.